CHANGELOG.md

```diff
@@ -1,5 +1,111 @@
 # Changelog
 
+## v6.0.0-beta.8 - 2017-10-05
+
+### Engine updates and fixes
+
+* Make calls to void CodeMethod work (#4850) (Thanks @powercode)
+* Get 'PSVersion' and 'GitCommitId' from the 'ProductVersion' attribute of assembly (#4863) (Thanks @iSazonov)
+* Write an error and return an exit code for powershell -version 2 (#4931) (Thanks @iSazonov)
+* Load the assembly with 'Assembly.LoadFrom' before 'Assembly.Load' when the file path is given (#4196)
+* Add a generic file watcher function in HelpersCommon.psm1 (#4775)
+* Update old links and fix broken links in host-powershell/README.md (#4877)
+* Fix importing remote module using version filters and added tests (#4900)
+* Enable transcription of native commands on non-Windows (#4871)
+* Fix `powershell -version` and help to align with other tools (#4958)
+* Fix incorrect position of a parameter which resulted in the args passed as input instead of as args (#4963)
+* Add a new line to 'CommandNotFoundException' error string (#4934 & #4991)
+
+### General cmdlet updates and fixes
+
+* Added Remove-service to Management module (#4858) (Thanks @joandrsn)
+* Added functionality to set credientials on `Set-Service` command (#4844) (Thanks @joandrsn)
+* `Select-String` changes:
+    * Exclude directories discovered from `-Path` (#4829) (Thanks @iSazonov)
+    * Support more argument completion scenarios by adding `ArgumentCompletionsAttribute` (#4835) (Thanks @iSazonov)
```

Comment:
This change is for `Get-Date`, not select-string. see https://github.com/PowerShell/PowerShell/pull/4835